src/pages/Booking/SeatBooking/BookSeats.js

```diff
@@ -0,0 +1,25 @@
+import Seats from './Seats';
+
+const createSection = numberOfRows => {
+  let curAlphabet = 'A';
+  const section = [];
+  for (let j = 0; j < numberOfRows; j++) {
+    for (let i = 0; i < 10; i++) {
+      if (j < numberOfRows + 1) {
+        section.push(curAlphabet + (i + 1));
+      }
+    }
+    curAlphabet = String.fromCharCode(curAlphabet.charCodeAt(0) + 1);
+  }
+  return section;
+};
+
+const BookSeats = ({ ticketCount, bookedSeats }) => {
+  const seats = createSection(10);
+
+  return (
+    <Seats values={seats} ticketCount={ticketCount} bookedSeats={bookedSeats} />
+  );
+};
```

Comment:
```suggestion
const BookSeats = ({ ticketCount, bookedSeats }) => {

const createSection = numberOfRows => {
  let curAlphabet = 'A';
  const section = [];
  for (let j = 0; j < numberOfRows; j++) {
    for (let i = 0; i < 10; i++) {
      if (j < numberOfRows + 1) {
        section.push(curAlphabet + (i + 1));
      }
    }
    curAlphabet = String.fromCharCode(curAlphabet.charCodeAt(0) + 1);
  }
  return section;
};


  const seats = createSection(10);

  return (
    <Seats values={seats} ticketCount={ticketCount} bookedSeats={bookedSeats} />
  );
};
```

함수컴포넌트 밖에 전역변수로 외롭게 두기보단, 컴포넌트 안쪽에서 선언해주시고 호출하는게 좋겠습니다.